artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2454,10 +2450,10 @@ public void run() {
 
                   if (doneCallback != null) {
                      doneCallback.run();
-                     doneCallback = null;
                   }
                }
 
+
```

Comment:
Empty line addition seems unecessary...but helpful to refer to the existing log message below it.
The _"Scanning for expires on " + QueueImpl.this.getName() + " done"_ message doesnt line up with when it calls the _doneCallback_, as it would be logged every time through, even after the rescheduling. It might be nice if the 'done' messaging did align as it would then work in concert with the other [suggested] logging to be much clearer on what is happening when.

Reply:
I wouldnt say this comment still applies. It seems strange to log that it is "done" in the cases it has just been logged that it isnt finished.